app/assets/javascripts/oxalis/store.js

```diff
@@ -93,7 +94,7 @@ export type TreeType = TreeTypeBase & {
   +nodes: NodeMapType,
 };
 
-type TemporaryMutableNodeMapType = { [number]: NodeType };
+type TemporaryMutableNodeMapType = DiffableMap<number, NodeType>;
```

Comment:
There no longer is a difference to the `NodeMapType`, might as well remove it?

Reply:
Good point, the type was ugly, anyway :)